"sails debug" on node v0.12.0 throwing error EADDRINUSE

Running Sails v0.11.0 and Node v0.12.0

When I run
```
sails new app; cd app
sails debug
```

Console logs...

```
error: Grunt :: Error: listen EADDRINUSE
    at exports._errnoException (util.js:746:11)
    at Agent.Server._listen2 (net.js:1129:14)
    at listen (net.js:1155:10)
    at Agent.Server.listen (net.js:1240:5)
    at Object.start (_debugger_agent.js:20:9)
    at startup (node.js:86:9)
    at node.js:814:3
```
No conflicting ports were listening prior to running sails.
Looking at travis.yml is only node v0.10.x supported currently?

I am running node -v 0.12.0
I get the following error 
Error: Cannot find module '/usr/local/lib/node_modules/node-inspector/node_modules/v8-debug/build/debug/v0.4.2/node-v14-darwin-x64/debug.node

on the console of the browser. when I run 

node-debug --debug-brk --harmony --use-strict app.js

on Mac terminal I get
--
Node Inspector is now available from http://127.0.0.1:8080/debug?port=5858
Debugger listening on port 5858

Starting the interactive shell (REPL). Type `.help` for help.
The module in the current directory was not loaded: Cannot find module '/Users/KK/git/DNetworksAuth'.
Didn't want to start REPL? Run `/usr/local/bin/node-debug .` instead.

I need to run node 0.11.x or above as I am using Web framework koa which needs it.

Please help.

--- a/lib/hooks/grunt/index.js
+++ b/lib/hooks/grunt/index.js
@@ -45,11 +45,6 @@ module.exports = function (sails) {
         taskName = '';
       }
 
-      var execArgs = process.execArgv.slice(0);
-      if(execArgs.indexOf('--debug') !== -1) {
-        execArgs.splice(execArgs.indexOf('--debug'),1);
-      }
-
       // Fork Grunt child process
       var child = ChildProcess.fork(
 
@@ -69,7 +64,11 @@ module.exports = function (sails) {
         {
           silent: true,
           stdio: 'pipe',
-          execArgv: execArgs
+          // pass all current node process arguments to the child process,
+          // except the debug-related arguments, see issue #2670
+          execArgv: process.execArgv.slice(0).filter(function (param) { 
+            return !(new RegExp('--debug(-brk=[0-9]+)?').test(param));
+          });
         }
       );
 

--- a/lib/hooks/grunt/index.js
+++ b/lib/hooks/grunt/index.js
@@ -68,7 +68,7 @@ module.exports = function (sails) {
           // except the debug-related arguments, see issue #2670
           execArgv: process.execArgv.slice(0).filter(function (param) { 
             return !(new RegExp('--debug(-brk=[0-9]+)?').test(param));
-          });
+          })
         }
       );
 

